front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -139,40 +139,41 @@ const ProjectsShowPage = ({ project }: Props) => {
         <ProjectHeader projectId={projectId} />
         <ProjectCTABar projectId={projectId} />
 
-        <div id="participation-detail">
-          <TimelineContainer projectId={projectId} />
-        </div>
-        {!!events?.data.length && (
-          <Box
-            id="e2e-events-section-project-page"
-            display="flex"
-            flexDirection="column"
-            gap="48px"
-            mx="auto"
-            my="48px"
-            maxWidth="1166px"
-            padding={isSmallerThanTablet ? '20px' : '0px'}
-          >
-            <EventsViewer
-              showProjectFilter={false}
-              projectId={projectId}
-              eventsTime="currentAndFuture"
-              title={formatMessage(messages.upcomingAndOngoingEvents)}
-              fallbackMessage={messages.noUpcomingOrOngoingEvents}
-              projectPublicationStatuses={['published', 'draft', 'archived']}
-            />
-            <EventsViewer
-              showProjectFilter={false}
-              projectId={projectId}
-              eventsTime="past"
-              title={formatMessage(messages.pastEvents)}
-              fallbackMessage={messages.noPastEvents}
-              projectPublicationStatuses={['published', 'draft', 'archived']}
-              showDateFilter={false}
-            />
-          </Box>
-        )}
-
+        <main>
```

Comment:
Only wraps the main content now. Container styles stayed in the same place. This component may still be up for improvement, but first finishing all WIP so things stay manageable.